src/thematic/0_generic/sfp-001_vervanging-terrest_nl/07_stappenplan.Rmd

```diff
@@ -0,0 +1,117 @@
+
+De eerste werkwijze wordt geïllustreerd in Figuur \@ref(fig:fig2) waarbij alle aanwezige doelhabitat wordt gekarteerd binnen de polygoon. Alle habitatvlekken van het doelhabitat worden dus in kaart gebracht. Bij voorkeur gebeurt deze kartering in de GIS-omgeving op de veldcomputer. Vervolgens kijk je welke potentiële vervangpunten in de gekaartede habitatvlekken vallen en selecteer je het vervangpunt met het laagste rangnummer. In het (fictieve) voorbeeld in Figuur \@ref(fig:fig2) selecteren we vervangpunt met rangnummer 3. Dit is het punt binnen het doelhabitat met het laagste relatieve rangnummer. 
+
+```{r fig2, fig.cap = "De geselecteerde steekproefeenheid (geel) is niet geschikt. Na kartering van het doelhabitat (gearceerde polygoon), wordt het vervangpunt met laagste rangnummer binnen het doelhabitat als vervangpunt geselecteerd"}
+
+knitr::include_graphics("media/vervangingsprocedure_1_stap2_metraster.png")
+
+```
+
+#### Wekwijze 2: evalueer de mogelijke vervangpunten in volgorde volgens rangnummer {#werkwijze2}
+
+Figuur \@ref(fig:fig3) illustreert een tweede mogelijkheid waarbij de potentiële vervangpunten in volgorde worden bezocht tot er een geschikt vervangpunt wordt gevonden. In het (fictieve) voorbeeld in Figuur\@ref(fig:fig3) wandel je dus eerst naar punt 1 (= niet geschikt), vervolgens naar punt 2 (= niet geschikt) en kom je vervolgens bij punt 3 dat wel geschikt blijkt te zijn. Als je onderweg in de buurt van andere punten passeert waarvan je snel kan inschatten dat ze niet in doelhabitat vallen, kan je dit uiteraard ook al noteren, zodat je daar niet meer moet terugkomen. Bijvoorbeeld, als je van punt 1 naar punt 2 wandelt, kan je mogelijks al zien dat punt 4 en 5 in een bosrand liggen. Dan kan je die punten al elimineren als mogelijke vervangpunten. 
+
+Deze werkwijze is minder aangewezen voor polygonen met een grote oppervlakte omdat de kans dan groot is dat je dan heel wat heen en weer moet wandelen. Uiteraard moeten beide mogelijke werwijzes tot het zelfde resultaat leiden.  
+
+```{r fig3, fig.cap = "De geselecteerde steekproefeenheid (geel) is niet geschikt. De potentiële vervangpunten worden in volgorde afgelopen en geëvalueerd tot een geschikt vervangpunt wordt gevonden"}
+
+knitr::include_graphics("media/vervangingsprocedure_1_stap2_bis_metraster.png")
+
+```
+#### Wat als er geen geschikt vervangpunt wordt gevonden?
+
+Wanneer er geen doelhabitat wordt gevonden in de polygoon zal er uiteraard geen vervangpunt kunnen geselecteerd worden. De vervangprocedure wordt afgerond en er wordt dus geen bemonstering uitgevoerd in de polygoon.
+
+Het kan ook voorkomen dat er wel doelhabitat aanwezig is, maar dat geen enkel vervangpunt (centroïde van een rastercel) in doelhabitat gelegen is. Dit komt voor in het (fictieve) voorbeeld in Figuur \@ref(fig:fig4). Ook in dit geval wordt er dus geen bemonstering uitgevoerd in de polygoon.
+
+*Moeten we voorzien dat er optioneel toch nog een verplaatsing kan gebeuren?*
```

Comment:
Absoluut. Merk op dat dit enkel zal voorkomen bij de centroïdgekoppelde types, niet bij de celgekoppelde.

In het [rapport van de doelpopulatieverkenning](https://drive.google.com/uc?export=download&id=1jH7rNSHvXycCiCrsoMRLaZ3IlY7GX35D) is het principe kort beschreven onder 8.3.2.1 (translatie). Met Thierry was al besproken hoe we dit kunnen doen, i.e. de centroïden verschuiven op verschillende gerandomiseerde manieren (komt neer op puntcoördinaten bijwerken, bv. beginnen met halverwege tussen bestaande punten, volgens bepaalde richting). Het selecteren van een dgl. vervangpunt gebeurt best geprogrammeerd, met als input het gekarteerde voorkomen van het doeltype. Manueel is dit niet te doen vrees ik.

Merk tenslotte ook op dat de gewone 'GRTS-toevoeging' op het einde van dat stappenplan vermeld wordt, om een vervangend punt te selecteren indien een polygoon het doeltype niet bevat. Dit hoort allicht ook op een of andere manier thuis in het vervangprotocol?

Reply:
OK. Ik voeg dat toe. Maar hoe je dat praktisch gaat aanpakken is mij nog zeer onduidelijk. Dus dat laat ik nog open.
In dat opzicht is werkwijze 2 ook af te raden bij centroïed gekoppelde types want dan moet je telkens opnieuw je punten aflopen.

Reply:
> hoe je dat praktisch gaat aanpakken is mij nog zeer onduidelijk

Ik denk dat het enkel te doen is met een applicatie. Dus niet op korte termijn haalbaar. Daarom akkoord dat we nog een ander alternatief nodig hebben, aangezien we een eenheid moeten selecteren als het doeltype aanwezig is.

> In dat opzicht is werkwijze 2 ook af te raden bij centroïed gekoppelde types

Indien het standaard raster geen punt oplevert, dan inderdaad af te raden ja. In het andere geval lijkt punten aflopen me wel de snelste methode.